h2o-automl/src/main/java/ai/h2o/automl/leaderboard/LeaderboardColumn.java

```diff
@@ -0,0 +1,16 @@
+package ai.h2o.automl.leaderboard;
+
+import hex.Model;
+import water.Freezable;
+import water.Key;
+
+public interface LeaderboardColumn<V, SELF extends LeaderboardColumn> extends Freezable<SELF> {
```

Comment:
@seb-h2o don't see usages for SELF, planning to add something?

Reply:
it's necessary to forward it to `Freezable`, and allow the implementations to extend `Iced`.
That's why I used the name `SELF`, to suggest that the implementation should put its own class here.